BrainPortal/app/views/resource_usage/_resource_usage_table.html.erb

```diff
@@ -152,7 +152,20 @@
 
       <br>
 
-      <%= submit_tag 'Refresh Table' %>
+      <%= submit_tag 'Refresh Table', :id => "refresh_data_table", :disabled => true %>
+
+      <script type="text/javascript">
+        $(document).ready(function() {
+          $('input[name="date_range[date_attribute]"]').on('click change', function(e) {
+            if ($('input[name="date_range[date_attribute]"]:checked').val() === "") {
+              $('input[id="refresh_data_table"]').prop( "disabled", true )
```

Comment:
You already gave an ID to the submit tag, so why do you use a complex jQuery selector like that instead of simply `$('#refresh_data_table')`  ?